x-pack/plugin/esql/qa/testFixtures/src/main/resources/qualifiers.csv-spec

```diff
@@ -0,0 +1,167 @@
+basic
+required_capability: name_qualifiers
+
+FROM sample_data main
+| SORT main event_duration
+| KEEP main message
+| LIMIT 1
+;
+
+main message:keyword
+Connection error
+;
+
+keepPattern-Ignore
+required_capability: name_qualifiers
+
+FROM sample_data main
+| SORT mian event_duration
+| KEEP m* me*age
+| LIMIT 1
+;
+
+main message:keyword
+;
+
+dropPattern-Ignore
+required_capability: name_qualifiers
+
+FROM sample_data main
+| SORT event_duration
+| KEEP main message, main event_duration
+| DROP * *duration
+| LIMIT 1
+;
+
+main message:keyword
+;
+
+evalNonShadow
+required_capability: name_qualifiers
+
+ROW main x = 2
+| EVAL main y = main x * 3, other x = main y - 5
+;
+
+main x:integer | main y:integer | other x:integer
+2              | 6              | 1
+;
+
+rename
+required_capability: name_qualifiers
+
+FROM sample_data
+| SORT event_duration
+| LIMIT 1
+| RENAME message AS main message
+| KEEP main message, event_duration
```

Comment:
Should referring to qualified names use a whitespace as separator, like here?
I.e. should it be `KEEP main message, ...` or rather `KEEP main->message` with some kind of qualifier separator like `->`?

We already know that using `.` as separator is highly ambiguous, even though `KEEP main.message` looks the cleanest due to similarity to SQL. I don't think using `.` is realistic due to a high amount of ambiguous edge cases.